sky/data/storage.py

```diff
@@ -3672,46 +3651,23 @@ def __init__(self,
         self.config_profile: str
         self.compartment: str
         self.namespace: str
-        # self.sema = Semaphore(5)
 
-        try:
-            # Bucket region should be consistence with the OCI config file
-            region = oci.get_oci_config()['region']
-        except (oci.oci.exceptions.ConfigFileNotFound,
-                oci.oci.exceptions.InvalidConfig, ImportError):
-            # For test only: In test env, the OCI config may not exist
-            region = 'us-sanjose-1'
+        # Bucket region should be consistence with the OCI config file
+        region = oci.get_oci_config()['region']
 
         super().__init__(name, source, region, is_sky_managed,
                          sync_on_reconstruction)
 
     def _validate(self):
         if self.source is not None and isinstance(self.source, str):
-            if self.source.startswith('s3://'):
-                assert self.name == data_utils.split_s3_path(self.source)[0], (
-                    'S3 Bucket is specified as path, the name should be the'
-                    ' same as S3 bucket.')
-                assert data_utils.verify_s3_bucket(self.name), (
-                    f'Source specified as {self.source}, an S3 bucket. ',
-                    'S3 Bucket should exist.')
-            elif self.source.startswith('gs://'):
-                assert self.name == data_utils.split_gcs_path(self.source)[0], (
-                    'GCS Bucket is specified as path, the name should be '
-                    'the same as GCS bucket.')
-                assert data_utils.verify_gcs_bucket(self.name), (
-                    f'Source specified as {self.source}, a GCS bucket. ',
-                    'GCS Bucket should exist.')
-            elif self.source.startswith('r2://'):
-                assert self.name == data_utils.split_r2_path(self.source)[0], (
-                    'R2 Bucket is specified as path, the name should be '
-                    'the same as R2 bucket.')
-                assert data_utils.verify_r2_bucket(self.name), (
-                    f'Source specified as {self.source}, a R2 bucket. ',
-                    'R2 Bucket should exist.')
-            elif self.source.startswith('oci://'):
+            if self.source.startswith('oci://'):
                 assert self.name == data_utils.split_oci_path(self.source)[0], (
                     'OCI Bucket is specified as path, the name should be '
                     'the same as OCI bucket.')
+            else:
```

Comment:
iiuc this source can be a local dir as well? and we should support it?

Reply:
Thanks. My bad! Introduced this mistake when address the comments.

Reply:
Well tested and updated the example yaml.